api/api/controllers/cluster_controller.py

```diff
@@ -80,8 +80,9 @@ async def get_cluster_nodes(request, pretty=False, wait_for_complete=False, offs
                           wait_for_complete=wait_for_complete,
                           logger=logger,
                           local_client_arg='lc',
-                          rbac_permissions=request['token_info']['rbac_policies'],
-                          nodes=nodes
+                          rbac_permissions=get_permissions(connexion.request.headers['Authorization']),
```

Comment:
Check this works correctly when [4300-dev-flask-rework-cluster-unittests ](https://github.com/wazuh/wazuh/pull/5095) is merged.

Reply:
```suggestion
                          rbac_permissions=request['token_info']['rbac_policies'],
```
We should use the `request` parameter instead.